boards/shared/Src/Io/Io_SharedCan.c

```diff
@@ -297,6 +298,10 @@ void Io_SharedCan_DequeueCanRxMessage(struct CanMsg *message)
         ;
 }
 
+// This extern is a hack for now. Need to rewrite
```

Comment:
Similar concern as: https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/615/files#r426098243.

There is no nice way to separate Error Table and CAN TX for now though.